docs/starters.yml

```diff
@@ -2935,3 +2935,12 @@
     - Styling with SCSS
     - Offline support
     - Web App Manifest
+- url: https://mystifying-khorana-1af5b2.netlify.com/
+  repo: https://github.com/benjamin-glitsos/gatsby-kea-starter
+  description: Gatsby starter with redux and sagas made simpler by the Kea library
+  tags:
+    - Kea
```

Comment:
Neither the tag `Kea` or the `Sagas` are not available. For more information, please have a look at this [link](https://www.gatsbyjs.org/starters/?v=2)